spec/json/twilio_trunking_v1.json

```diff
@@ -2008,7 +1998,27 @@
           "content": {
             "application/x-www-form-urlencoded": {
               "schema": {
-                "properties": {},
+                "properties": {
```

Comment:
These changes probably shouldn't be in here. Double check api-definitions hash

Reply:
My bad

Reply:
Need to do a `git rm .DS_Store` (the file is now ignored, but we need to remove it from this PR also)

Reply:
Done